app/models/katello/concerns/location_extensions.rb

```diff
@@ -58,6 +59,19 @@ def deletable?
         end
       end
 
+      def associate_default_http_proxy
+        setting = Setting::Content.where(name: 'content_default_http_proxy').first
```

Comment:
Nitpick, but this is a bit more concise: 
find_by(name: 'content_default_http_proxy')